docs/about-core-concepts/architecture-guide.rst

```diff
@@ -22,20 +22,24 @@ be indexed into the ODC, so long as it can be described by ODC metadata.
 This section of the documentation describes the structure of the ODC and the key
 components that make up an implementation, as well as the ecosystem around it.
 
+.. figure:: ../diagrams/odc1.9-arch.png
+   :name: high-level-overview
+
+   Datacube-core vs odc-stac
+
 .. _`Cloud Optimised GeoTIFFs`: https://www.cogeo.org/
 .. _`SpatioTemporal Asset Catalog`: https://stacspec.org/
 .. _`Sentinel-2 Indexing notes`: https://github.com/opendatacube/datacube-dataset-config/blob/master/sentinel-2-l2a-cogs.md
 .. _`Datacube Explorer`: https://github.com/opendatacube/datacube-explorer
 
 
-Database Diagram
-----------------
-
-.. figure:: ../diagrams/db-relationship-diagram.svg
-    :target: /_images/db-relationship-diagram.svg
-
-    Current database relationship diagram
+Metadata Conceptual Diagram
```

Comment:
Perhaps this belongs in the metadata types page rather than the overview? I think this diagram introduces concepts (like locations and lineage) that aren't described in the current text, which may confuse the reader.